eta/core/primitives.py

```diff
@@ -148,7 +164,7 @@ def __init__(
         self._flags = (
             cv2.OPTFLOW_FARNEBACK_GAUSSIAN if use_gaussian_filter else 0)
 
-    def _process_frame(self, img):
+    def process_frame(self, img):
```

Comment:
@brimoor since this function only return the output in cartesian format, is your idea to transform the output to the other two formats(polar and visualization image) in the rapid motion detection module.

Reply:
I refactored this functionality into a standalone method so that the user of this class can convert their Cartesian flow fields to polar if they want to.

The reason I don't return all formats (cartesian, polar, and HSV image) by default is that the caller might not want all of these formats, so we should avoid doing any unnecessary computation. We should be especially careful about avoiding unnecessary computation in `core` modules